tests/test_opSimOrdering.py

```diff
@@ -0,0 +1,76 @@
+import os
+import numpy as np
+import pandas as pd
+from desc.twinkles import OpSimOrdering
+from lsst.utils import getPackageDir
+import unittest
+
+class TestOpSimOrdering(unittest.TestCase):
+
+    def setUp(self):
+        self.opSimDBPath = '/Users/rbiswas/data/LSST/OpSimData/minion_1016_sqlite.db'
```

Comment:
If we use a tiny opsim file as we discussed in the other comment, then set the path by doing
```
self.opSimDBPath = os.path.join(getPackageDir('Twinkles'), 'data', <small opsimdb filename>)
```
we can avoid the extra setup step.